The error is listed below, though I am not sure why it's happening, since you imported logging already.

```
INFO: From Testing //tensorflow/python:flags_test:
==================== Test output for //tensorflow/python:flags_test:
Traceback (most recent call last):
  File "/var/lib/jenkins/workspace/tensorflow-pull-requests-cpu/bazel-ci_build-cache/.cache/bazel/_bazel_jenkins/eab0d61a99b6696edb3d2aff87b585e8/execroot/workspace/bazel-out/local-opt/bin/tensorflow/python/flags_test.runfiles/org_tensorflow/tensorflow/python/platform/flags_test.py", line 117, in <module>
    flags.mark_flag_as_required('string_foo_required')
  File "/var/lib/jenkins/workspace/tensorflow-pull-requests-cpu/bazel-ci_build-cache/.cache/bazel/_bazel_jenkins/eab0d61a99b6696edb3d2aff87b585e8/execroot/workspace/bazel-out/local-opt/bin/tensorflow/python/flags_test.runfiles/org_tensorflow/tensorflow/python/platform/flags.py", line 173, in mark_flag_as_required
    logging.warning(
NameError: global name 'logging' is not defined
```

--- a/tensorflow/python/platform/flags.py
+++ b/tensorflow/python/platform/flags.py
@@ -20,7 +20,7 @@ from __future__ import print_function
 
 import argparse as _argparse
 
-from tensorflow.python.platform import tf_logging as logging
+from tensorflow.python.platform import tf_logging as _logging
 from tensorflow.python.util.all_util import remove_undocumented
 
 _global_parser = _argparse.ArgumentParser()
@@ -170,7 +170,7 @@ def mark_flag_as_required(flag_name):
       NOTE: The exception raised will change in the future. 
   """
   if _global_parser.get_default(flag_name) is not None:
-    logging.warning(
+    _logging.warn(
         'Flag %s has a non-None default value; therefore, '
         'mark_flag_as_required will pass even if flag is not specified in the '
         'command line!' % flag_name)

